internal/hcsoci/hcsdoc_wcow.go

```diff
@@ -67,6 +67,9 @@ func createMountsConfig(ctx context.Context, coi *createOptionsInternal) (*mount
 					return nil, err
 				}
 				mdv2.HostPath = uvmPath
+			} else if strings.HasPrefix(mount.Source, "sandbox://") {
```

Comment:
I think you need to rebase this to take in the changes for adding ExtensibleVirtualDisk mounts that I just merged. Haha.

Reply:
Thanks for the heads up! Let me do that now

Reply:
Done!

Reply:
you'll need some changes in cri for this to work too, particularly [here](https://github.com/jterry75/cri/blob/df7756eb3adfd39c413f099c841a2466e757a491/pkg/server/container_create_windows.go#L511) if you haven't already taken a look at that :)